docs/source/usage/python.rst

```diff
@@ -375,7 +378,8 @@ Particles
       :param py: momentum in y
       :param pt: momentum in t
       :param qm: charge over mass in 1/eV
-      :param bchchg: total charge within a bunch in C
+      :param bunch_charge: total charge within a bunch in C
+      :param w: weight of each particle: how many real particles to represent
```

Comment:
What if we call this something like `macroparticle_charge`:  the macroparticle charge in units of the elementary charge `e`.